docs/installation/config/cmis.rst

```diff
@@ -99,6 +99,116 @@ Using the CMIS adapter
 
 .. _`CMIS adapter library`: https://github.com/open-zaak/cmis-adapter
 
+When using the CMIS-adapter, sometimes a Zaak, a Zaaktype or a Besluit need to be retrieved from the APIs.
+This happens for example when a Zaak folder is created, since the description of the Zaak is used as part of the folder name.
+
+For this reason, the CMIS-adapter needs to be given access to the various APIs.
+These are the steps to configure it:
+
+1. Go to **API Autorisaties > Applicaties** and click on **Applicatie toevoegen**.
+
+2. Enter a human readable label, for example ``CMIS adapter``.
+
+3. Enter a **Client ID**, for example ``cmis-adapter``.
+
+4. Enter a **Secret**. This should be a random string, for example ``Tl8@04&O4gXTtB``.
+
+5. Click **Opslaan en opnieuw bewerken**.
+
+6. Click **Beheer Autorisaties**.
+
+7. Under **Component** check ``Zaken API``, under **Selecteer scopes** check ``zaken.lezen``, under **Voor welke typen geldt dit?** check ``Alle huidige en toekomsitge ZAAKTYPEN`` and  under **Tot en met welke vertrouwelijkheidaanduiding?** check ``Zeer geheim``.
+
+8. Click on **Nog Autorisaties toevoegen** and under **Component** check ``Catalogi API`` and under **Selecteer scopes** check ``catalogi.lezen``.
+
+9. Click on **Nog Autorisaties toevoegen** and check under **Component** ``Besluiten API``, under **Selecteer scopes** check ``besluiten.lezen`` and under **Voor welke typen geldt dit?** check ``Alle huidige en toekomsitge BESLUITTYPEN``.
+
+10. Click on **Nog Autorisaties toevoegen** and under **Component** check ``Documenten API``,  under **Selecteer scopes** check ``documenten.lezen``,  under **Voor welke typen geldt dit?** check ``Alle huidige en toekomsitge INFORMATIEOBJECTTYPEN`` and under **Tot en met welke vertrouwelijkheidaanduiding?** check ``Zeer geheim``.
+
+11. Click on **Opslaan** to save.
+
+Then, the services need to be configured.
+First, configure the Zaken API as follows:
+
+1. Go to **API Autorisaties > Services** and click on **Service Toevoegen**.
+
+2. Add a human readable label, for example ``Zaken API``.
+
+3. Change the type of the API to ``ZRC (Zaken)``.
+
+4. Enter the API root of the Zaken API, for example ``http://example.com/zaken/api/v1/``.
+
+5. Enter the **Client ID**. Use the one configured in the **API Autorisaties > Applicaties**. Following the previous example, this would be ``cmis-adapter``.
+
+6. Enter the **Secret**. Use the one configured in the **API Autorisaties > Applicaties**. Following the previous example, this would be ``Tl8@04&O4gXTtB``.
+
+7. Leave the **Authorization type** as ``ZGW client_id + secret``.
+
+8. Add the link to the **OAS**. For example, ``http://example.com/zaken/api/v1/schema/openapi.json``.
```

Comment:
Can you replace these steps with "Fill out the form"  like done here: https://open-zaak.readthedocs.io/en/latest/installation/config/openzaak_config.html#open-zaak